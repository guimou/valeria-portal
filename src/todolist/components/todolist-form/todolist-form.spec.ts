import { genSnapshotsTests, PropsData } from '@jest/jest-utils';
import TodolistForm from './todolist-form';

const states: PropsData = {
    'new state': {
        state: {
            mode: 'new',
            open: false
        } as Todolist.TodolistFormState
    },
    'edit state': {
        state: {
            mode: 'edit',
            open: false
        } as Todolist.TodolistFormState
    }
};

describe('TodolistForm', () => {
    genSnapshotsTests(TodolistForm, states);
});
