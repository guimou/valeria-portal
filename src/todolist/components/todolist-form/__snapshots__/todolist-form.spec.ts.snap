// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TodolistForm edit state 1`] = `
<m-modal title="Edit">
  <section>
    Title: <m-textfield-stub type="text" passwordicon="true" maxlength="0" lengthoverflow="true" charactercountthreshold="0"></m-textfield-stub> <br>
    Description: <m-textfield-stub type="text" passwordicon="true" maxlength="0" lengthoverflow="true" charactercountthreshold="0"></m-textfield-stub>
    <m-button>Ok</m-button>
    <m-button>Cancel</m-button>
  </section>
</m-modal>
`;

exports[`TodolistForm new state 1`] = `
<m-modal title="New">
  <section>
    Title: <m-textfield-stub type="text" passwordicon="true" maxlength="0" lengthoverflow="true" charactercountthreshold="0"></m-textfield-stub> <br>
    Description: <m-textfield-stub type="text" passwordicon="true" maxlength="0" lengthoverflow="true" charactercountthreshold="0"></m-textfield-stub>
    <m-button>Ok</m-button>
    <m-button>Cancel</m-button>
  </section>
</m-modal>
`;
